src/scripts/infuse/infuse.html

```diff
@@ -18,34 +18,34 @@ <h1 translate="Infusion"></h1>
       </div>
 
       <div class="infuseControls">
-        <label><input class="controls" type="checkbox" id="infuse-all" ng-model="vm.getAllItems" ng-change="vm.getItems()"><span translate="Infusion.ShowItems"></span></label>
-        <label><input class="controls" type="checkbox" id="infuse-locked" ng-model="vm.showLockedItems" ng-change="vm.getItems()"><span translate="Infusion.LockedItems"></span></label>
+        <label><input class="controls" type="checkbox" id="infuse-all" ng-model="vm.getAllItems" ng-change="vm.getItems()"><span ng-i18next="Infusion.ShowItems"></span></label>
+        <label><input class="controls" type="checkbox" id="infuse-locked" ng-model="vm.showLockedItems" ng-change="vm.getItems()"><span ng-i18next="Infusion.LockedItems"></span></label>
       </div>
-    </div>
```

Comment:
Something happened with the HTML in this page - I think it's missing a tag now (or nested differently).